assets/components/paymentButtons/payPalExpressButton/payPalExpressButton.jsx

```diff
@@ -56,24 +56,16 @@ function Button(props: PropTypes) {
     props.currencyId,
     props.csrf,
     props.callback,
-  );
-
-  const disabledButton = (
-    <button
-      className="component-paypal-button-checkout__disabled-pop-up-button"
-      disabled
-    >
-      Pay with PayPal
-      <SvgArrowRightStraight />
-    </button>
+    props.canOpen,
+    props.whenUnableToOpen,
   );
 
 
   const ActiveButton = window.paypal.Button.driver('react', { React, ReactDOM });
```

Comment:
I guess this can just be called `PayPalButton` now?

Reply:
It's still distinct from the one-off PayPal button isn't it?